### DISCUSSION TITLE
fop discussion

### DISCUSSION THREAD

**Discussion by Ketishavadze** [Hikarunnie/FOP-project#3] [General]
should we start dividing our roles and take responsibility?

**Comment by Hikarunnie** [comment]
review and accept my pull request first
@Ketishavadze @iblameluka @Mariammm870

**Comment by Mariammm870** [comment]
should we be using recursion?

**Comment by Hikarunnie** [comment]
huh, why?

**Reply by Ketishavadze** [reply]
I think it was written in the rules

**Reply by Hikarunnie** [reply]
ხო მაგრამ, ფიბონაჩისთვის რომ გვექნება დასაწერი, რეკურსიითაც შეგიძლიათო და რანაირად

**Comment by lookachu** [comment]
I think it's allowed
I mean... if it's not we can think about it tomorrow

**Reply by Hikarunnie** [reply]
Valid

**Comment by Ketishavadze** [comment]
okay now time to actually think more precisely who is working on what part of the code, please discuss and decide it

**Reply by Hikarunnie** [reply]
we have to add expression evaluation method other operations, we only have addition. we also need code for if else, we need for loop, we need while loop ... we can devide these between each other

**Reply by Hikarunnie** [reply]
lets decide quickly to finish working

**Reply by lookachu** [reply]
ok I'll handle the basic operations (+, -, *, /, %)

**Reply by Mariammm870** [reply]
I will try to write code for while loop.

**Reply by Hikarunnie** [reply]
alright , I will work on if else and keti can write for loop okay? # @Ketishavadze

**Reply by Ketishavadze** [reply]
Sure, no problem!

**Comment by Hikarunnie** [comment]
btw axla vamtavreb ifelses da evaluate_condition mwirdeba conditionis dasadgenad. wesit while loopshic dagwirdeba mari, da vinmes tu gindat aiget an me davwer np

**Reply by Mariammm870** [reply]
ukve dawerili maqvs, cota xanshi avtirtav da shegidzlia gamoiyeno

**Comment by Hikarunnie** [comment]
okay yall lets meet up 2morrow and finish working on code. then split our code parts to add comments aboput their functionality and finish writing README file alr?

**Reply by lookachu** [reply]
sure

**Comment by Hikarunnie** [comment]
@Ketishavadze  did you finish writing for loop? we need it to test few algorithms

**Reply by Ketishavadze** [reply]
Yes, I will upload it shortly.

**Comment by Hikarunnie** [comment]
this if else is hurting my head someone help T_T

**Comment by Hikarunnie** [comment]
yall i wanted to test it additionally on other algorithms like odd or even check, it does work which makes me happy ^ - ^ 
PS . thing is it can't handle print( variable + " String" ) type of printing operation,  ill add that function (if i'm not too lazy :)))))

**Reply by Hikarunnie** [reply]
about this I did write it but it made the code overly complicated so I left it as it was

**Comment by Mariammm870** [comment]
we should all write description for our functions in README.md file, thats the last thing we have left to do

**Comment by Hikarunnie** [comment]
ppl I deleted most of the comments we had coz I thought we weren't supposed to write them but apparently we need to add them.... T_T

**Reply by Mariammm870** [reply]
okay we can add them

**Comment by Hikarunnie** [comment]
@iblameluka i changed few things with the text u gave me for ReadMe file and uploaded it since you couldn't, do u want me to change anything else?

**Reply by lookachu** [reply]
no no, thanks so much <333
